gno.land/pkg/sdk/vm/msgs.go

```diff
@@ -75,19 +76,20 @@ func (msg MsgAddPackage) GetSigners() []crypto.Address {
 
 // Implements ReceiveMsg.
 func (msg MsgAddPackage) GetReceived() std.Coins {
-	return msg.Deposit
+	return msg.Send
 }
 
 //----------------------------------------
 // MsgCall
 
 // MsgCall - executes a Gno statement.
 type MsgCall struct {
-	Caller  crypto.Address `json:"caller" yaml:"caller"`
-	Send    std.Coins      `json:"send" yaml:"send"`
-	PkgPath string         `json:"pkg_path" yaml:"pkg_path"`
-	Func    string         `json:"func" yaml:"func"`
-	Args    []string       `json:"args" yaml:"args"`
+	Caller     crypto.Address `json:"caller" yaml:"caller"`
+	Send       std.Coins      `json:"send" yaml:"send"`
+	MaxDeposit std.Coins      `json:"max_deposit,omitempty" yaml:"max_deposit"`
```

Comment:
should we perform the same check as in addpkg to be consistent
i mean the MaxDeposit.IsValid ?
same for the MsgRun

Reply:
@MikaelVallenet You're right. We should definitely add that check. I’ll put together a PR to patch these validations.

Reply:
@MikaelVallenet  

Here is the patch PR
https://github.com/gnolang/gno/pull/4512